components/Modal.js

```diff
@@ -0,0 +1,37 @@
+import { changeData } from '../api/requests.js';
+
+export class Modal {
+  modalWindowContainer = null;
+  constructor(modalWindowContainer) {
+    this.modalWindowContainer = modalWindowContainer;
+  }
+
+  createModal(matchObj, rowId) {
+    let changedDataObj = {};
+    for (let key in matchObj) {
+      if (key !== 'id') {
+        const inputField = document.createElement('input');
+        inputField.classList.add('modal-input-flied');
+        inputField.placeholder = key;
+        inputField.value = matchObj[key];
+        changedDataObj[key] = inputField.value;
+
+        inputField.addEventListener('input', () => {
+          changedDataObj[key] = inputField.value;
+        });
```

Comment:
Let's create a separate method, that will render our data
In the scope of moving your modal initialization in the `Modal`, please do it